.vs/ProjectSettings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "CurrentProjectSetting": null
+}
```

Comment:
Let's start with adding the blank line at the end of each file :)  (POSIX standard)